flang/include/flang/Runtime/character.h

```diff
@@ -6,6 +6,37 @@
 //
 //===----------------------------------------------------------------------===//
 
+// Defines a utility function for copying and padding characters
+#ifndef CHARACTER_H
+#define CHARACTER_H
```

Comment:
Did you intend to place this code in a new runtime/character.h header?

It would make more sense to me to no expose this helper in the public runtime API headers, I think you could just put in in runtime/tools.h that is a good fit for it.

Reply:
Either way, the name of the guard macro is wrong.

Reply:
Thanks for pointing this out, I have moved it under `runtime/tools.h`.